README.md

```diff
@@ -54,6 +54,74 @@ to you z config:
 messages:
     overwrite_compatibility: false
 ```
+
+## Automatic translation
+Leverage the `zicht:message:translate` command with automatic translation through an API of choice.
+Kickstart the usage by using the provided Google Translate API.
+
+### Setup
+
+`composer require google/cloud-translate`
+
+### Configuration
+Define the Google Translator as a service in your project. Your API key should be the private key from a service account (https://cloud.google.com/translate/docs/basic/setup-basic)
+
+```yaml
+parameters:
+    env(GOOGLE_API_KEY): '%kernel.root_dir%/config/your-google-api-key.json'
+```
+```xml
+<service id="Zicht\Bundle\MessagesBundle\Translator\GoogleTranslator" class="Zicht\Bundle\MessagesBundle\Translator\GoogleTranslator">
+    <argument key="$googleTranslateServiceAccount">%env(json:file:resolve:GOOGLE_API_KEY)%</argument>
+</service>
+```
+
+Add a `CompilerPass` to register the service on the `TranslateCommand`:
+
+```php
+/**
+ * @{inheritDoc}
+ */
+public function process(ContainerBuilder $container)
+{
+    $container
+        ->getDefinition(TranslateCommand::class)
+        ->addMethodCall('setBatchTranslator', [new Reference(GoogleTranslator::class)]);
+}
+```
+
+### Usage
+
+In this example we have copied a `.nl.yaml` to a `es.yaml` and we are informing the command that the sourcelanguage is `nl` and the
+targetlanguage should be `es`. As we have already renamed the file, only contents of this file will be rewritten.
+
+```shell script
+php bin/console zicht:message:translte /dir/to/project/translations/validators.es.(yaml|xlf) --source=nl --target=es
```

Comment:
`translte` -> `translate`

Reply:
`/dir/to/project/translations/validators.es.(yaml|xlf)` is de source file, waarom is er dan nog een --source property nodig?

Zou het (kwa interface) simpeler zijn om 2 argumenten te hebben, (1) een source file en (2) een target file.  Middels een regexp kan dan de source language en target language uit de filename gehaald worden

Reply:
De filename kan ook `validator.es-MX' zijn, of `nl-BE`, dan wordt het uitvogelen van een locale alweer ingewikkelder.
Daarbij, (praktisch voorbeeld) voor Noors kun je `nn`, gebruiken maar de Google API verwacht `no` (ze zijn beide officiele locales). Dus hoe krijg je dan `no` naar de API gestuurd.

Ik hoef niet perse vast te zitten aan een bestandsnaam. Deze interface maakt het eenvoudig om onafhankelijk van wat je bestandsnaam ook is een bestand te laten vertalen door te zeggen om welk bestand het gaat en een `van` -> `naar` op te geven.